rBoot_Generic/readme.txt

```diff
@@ -0,0 +1,46 @@
+rBoot Generic example
+
+This is an alpha implementation of a Generic rBoot layer for easing the use of rBoot and provide a rBoot Bios application to support ESP fully remote.
+
+This PR includes a "RBootClass" which implements :
+
+    An initial layer around rBoot/rbootHttpUpdate
+    An initial way to mount/unmount spiffs filesystems, based on rBoot configuration
+    An initial CommandHandler interface to the functionality.
+    An example application using the above : rboot_generic
+
+Not much documentation yet,
+
+    see RBootClass.h for usage & interfaces
+    connect with telnet client to esp, rboot commands are then available : "rboot info" "rboot load 1 http://10.0.0.201/rom0.bin" "rboot ls" etc type "rboot" for all options.
+
+All command examples have equivalent API's -> available within application
+
+Current V0.1 is only usable/has hardcoded values for use on 4Mb (esp-12) devices, no other supported yet.
+
+The used romconfig is the following :
+
+ROM0 -> 0x002000	-> number from initial rboot figures, application area
+ROM1 -> 0x202000	-> number from initial rboot figures, application area
+ROM2 -> 0x100000;	-> number set in application code, spiffs area, size 65536
```

Comment:
Referring to spiffs files as roms is confusing, I'd avoid that. They are treated differently by rBoot (i.e. rBoot doesn't know about them), and have different purposes. I'd refer to them as spiffs0 & spiffs1.